src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.ts

```diff
@@ -127,7 +168,149 @@ export const transformOktaVerifyEnrollPoll: IdxStepTransformer = ({ transaction,
       } as DescriptionElement,
     ],
   });
-  if (deviceBootstrap) {
+  if (deviceMap.setupOVUrl) {
+    if (deviceMap.isDesktop) {
+      sameDeviceOVElements.push(
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            contentClassname: 'desktop-instructions ov-info',
+            content: loc(
+              'oie.enroll.okta_verify.setup.customUri.prompt',
+              'login',
+              [],
+              { $1: { element: 'span', attributes: { class: 'strong' } } },
+            ),
+          },
+        } as DescriptionElement,
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            content: loc(
+              'oie.enroll.okta_verify.setup.customUri.noPrompt',
+              'login',
+              [],
+              { $1: { element: 'span', attributes: { class: 'strong' } } },
+            ),
+          },
+        } as DescriptionElement,
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            content: loc('oie.enroll.okta_verify.setup.customUri.makeSureHaveOV', 'login'),
+          },
+        } as DescriptionElement,
+      );
+    } else {
+      sameDeviceOVElements.push(
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            contentClassname: 'desktop-instructions ov-info',
```

Comment:
Is this intentional? The `DescriptionElement` doesn't have a `contentClassname` in `options`

Reply:
@trevoring-okta please see updated PR with the second commit that cleans up this class name